src/app/components/publiclists/teamlist/teamcard/teamcard.component.ts

```diff
@@ -13,6 +14,21 @@ import { Router, ActivatedRoute } from '@angular/router';
 })
 export class TeamcardComponent implements OnInit, OnChanges {
 
+  /**
+   * User Name
+   */
+  user = {username: ''};
```

Comment:
Please change the variable to `LoggedInUser` or something more sensible and also change the comment to `LoggedIn User Details`